tools/prepend-type-ref.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @fileoverview Prepends a TypeScript reference comment to the beginning of a file.
+ * This is necessary because JSR requires that all JavaScript files have a reference
+ * to the TypeScript types file. We can't do this in Rollup because that happens
+ * before tsc is run. This script is run after tsc is run.
+ *
+ * Usage:
+ *  node scripts/prepend-type-ref.js filename.js
+ *
+ * @author Nicholas C. Zakas
+ */
+/* global process */
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
+import fs from "node:fs";
+import path from "node:path";
+
+//-----------------------------------------------------------------------------
+// Main
+//-----------------------------------------------------------------------------
+
+// read file from the command line
+const filePath = process.argv.slice(2)[0];
+const filename = path.basename(filePath, ".js");
+
+// read the file
+const contents = fs.readFileSync(filePath, "utf8");
+
+// prepend the reference comment
+const newContents = `/// <reference types="./${filename}.d.ts" />\n${contents}`;
```

Comment:
Can we remove the contents of `types.ts` here? This will make the package smaller

Reply:
I meant to delete its contents and keep only the triple-slash directive. not delete this file